crates/bevy_pbr/src/light.rs

```diff
@@ -92,6 +92,9 @@ impl Default for PointLightShadowMap {
 /// approximation for light sources VERY far away, like the sun or
 /// the moon.
 ///
+/// By default, the light shines along the negative-Z axis and the local
+/// transform adjusts its direction.
```

Comment:
```suggestion
/// transform adjusts its direction.
/// The light always points in the "forward" direction of its local rotation.
```

Reply:
The problem with this is that people don’t always know that negative z is forward.

Reply:
It's explained above, so I think this is fine.

Reply:
Oh, I misread the diff in the email and thought you were replacing the text. I see now you added to it. Apologies.